### DISCUSSION TITLE
Precision Range Sliders (mobile)

### DISCUSSION THREAD

**Discussion by luwes** [muxinc/media-chrome#421] [Proposals]
Media Chrome's time range is already very good compared to some players but there's still room for improvement.
Especially mobile needs some attention, it's a bad UX sometimes dragging the thumb...

It should work more or less like in iOS 16 new video player
https://9to5mac.com/2022/06/23/ios-16-video-player/

Related experiment https://twitter.com/frantzfries/status/1611505023556485120

**Comment by heff** [comment]
Nice. I like the direction. 👍
